src/components/component.jade

```diff
@@ -25,7 +25,7 @@ block root
       block script
     
     script.
-      if(window['params'].monitoring){
+      if(window['params'].monitoring!='false' && window['params'].monitoring!=null){
```

Comment:
Maybe we should have it just check == 'true'?

Reply:
The other usecase is user can input different GA key, so if they set monitoring=UA-xxx it will not pass this condition (if monitoring=='true'). So, the required condition to run the GA script is either:
- monitoring is true or monitoring contains GA key
or the one that I set
- monitoring not false and monitoring not empty
the lack of the second condition is if they put some random strings it will still run the script. Although it will not break the application, but it is not correct. I'm not sure if we should add a checker to check if it is a correct GA key though. We can give this responsibility to the user so they SHOULD put a correct GA key or the monitoring on their side won't work.

Reply:
Ok I get you 👍